routes/user.js

```diff
@@ -20,10 +20,11 @@ router.use('/:id/reviews', reviewRouter)
 router.get('/', asyncWrapper(userController.getUsers))
 router.get('/:id', asyncWrapper(userController.getOneUser))
 router.get('/my-profile', setCurrentUserIdAndRole, asyncWrapper(userController.getOneUser))
+router.patch('/me', setCurrentUserIdAndRole, asyncWrapper(userController.updateUserProfile))
```

Comment:
totally disagree with renaming update-my-profile to 'me'
why?
because you have /my-profile endpoint that returns actually "me"
if anything you would have to rename both of them
to me and update-me respectively, but since we stick to 'my-profile', then everything else must follow the same pattern:

update-my-profile
delete-my-profile
etc etc

+ as i already said, you MUST arrange all endpoints following some logic.
i would move all 'my-profile' related endpoints before 20 line and separate them with an empty row or would make an empty row after 21 line

Reply:
yeah, you are right